scripts/scripts.js

```diff
@@ -783,6 +783,87 @@ const preflightListener = async () => {
   customModal.showModal();
 };
 
+// Set event for the publish button for confirmation message
+const publishConfirmationPopUp = (oSidekick, oPublishButtons) => {
+  // Add plugin listeners here
+  if (!oSidekick) {
+    return;
+  }
+  oSidekick.addEventListener('custom:preflight', preflightListener);
```

Comment:
@jjjaspher looks like you're forcing the `preflight` option to show only when the user clicks on the publish, please check and fix.

Reply:
hi @rpapani , preflight will run once the publish button is already exist on the markup and not waiting for the click event.